tests/integration_tests.bzl

```diff
@@ -0,0 +1,22 @@
+load("@io_bazel_rules_go//go/tools/bazel_testing:def.bzl", "go_bazel_test")
+
+def integration_test(name, **kwargs):
+    test_src = name + ".go"
+    size = kwargs.pop("size", "medium")
+
+    native.config_setting(
+        name = "nixpkgs",
+        values = {"define": "nixpkgs=true"},
+    )
```

Comment:
I think you could reuse [this `config_setting`](https://github.com/tweag/rules_haskell/blob/7675f021fb2bb64d21f4b1c6bb3f77e02ceb8c83/tests/BUILD.bazel#L39-L44) based on the `support_nix` platform constraint. Then you also don't need to introduce the `--define=nixpkgs` above.

Reply:
Thanks! Fixed